src/modules/keyboardmanager/common/KeyboardManagerState.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <map>

```

Comment:
Duplicate comment:  I"m not sure you need *all* the includes in the header file.  This can slow down build times.